src/composite_algs.jl

```diff
@@ -10,17 +10,17 @@ mutable struct AutoSwitch{nAlg,sAlg,tolType,T}
   dtfac::T
   stiffalgfirst::Bool
 end
-AutoSwitch(nonstiffalg::nAlg, stiffalg::sAlg;
+AutoSwitch(nonstiffalg, stiffalg;
            maxstiffstep=10, maxnonstiffstep=3,
-           nonstifftol::T=9//10, stifftol::T=9//10,
-           dtfac=2.0, stiffalgfirst=false) where {nAlg,sAlg,T} =
+           nonstifftol=9//10, stifftol=9//10,
+           dtfac=2.0, stiffalgfirst=false) =
```

Comment:
Seems a bit strange that the default for `dtfac` is 2.0 rather than 2 (i's the same in OrdinaryDiffEq). Is there a reason for not taking an integer as default value?

Reply:
Yeah, I should change it